equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm4Controller.java

```diff
@@ -22,15 +24,23 @@ public class Crm4Controller implements Crm4InterfaceApi {
     private final Crm4Mapper crm4Mapper;
 
     @Override
-    public ResponseEntity<Crm4DetailsDTO> getApplicationCrm4(Long usn, String profileAcceptedTypes) {
+    public ResponseEntity<Crm4FormDTO> getApplicationCrm4(Long usn, String profileAcceptedTypes) {
         log.info("eForm CRM4 details request received :: usn=[{}]", usn);
         CrmFormDetailsCriteriaDTO crmFormDetailsCriteriaDTO = new CrmFormDetailsCriteriaDTO(
                 usn, CRM_TYPE_4, profileAcceptedTypes
         );
-        Crm4DetailsModel crm4FileDetails = (Crm4DetailsModel) crmFileService
-                .getCrmImageFile(crmFormDetailsCriteriaDTO)
-                .getFormDetails();
+        Crm4Model crm4FormData = crmFileService.getCrmImageFile(crmFormDetailsCriteriaDTO);
+
+        Crm4DetailsModel crm4FileDetails = crm4FormData.getFormDetails();
         crm4FileDetails.setAllQuotes();
-        return ResponseEntity.ok(crm4Mapper.getEntityFromModel(crm4FileDetails));
+
+        CrmEvidenceFilesModel evidenceFiles = crm4FormData.getEvidenceFiles();
+
+        Crm4FormDTO returnDTO = new Crm4FormDTO();
```

Comment:
` crm4FormReturnDTO` - Can you rename it?

Reply:
I would like to keep this variable name type/class agnostic. So it can easily be ported to the other different CRM Form controllers. The class name already specify what it is.

Reply:
will change to something more descriptive yet generic. like crmFormDTO